btcutil/psbt/partial_output.go

```diff
@@ -144,8 +146,23 @@ func (po *POutput) deserialize(r io.Reader) error {
 			)
 
 		default:
-			// Unknown type is allowed for inputs but not outputs.
-			return ErrInvalidPsbtFormat
+			// A fall through case for any proprietary types.
+			keyintanddata := []byte{byte(keyint)}
```

Comment:
Should use camel case for `keyintanddata` and the other variables.

Reply:
Oops, copy/paste. Fixed it (and some other formatting issues) in the existing code too.